web-wallet/src/lib/stores/__tests__/walletStore.spec.js

```diff
@@ -1,61 +1,31 @@
-import { afterAll, beforeEach, describe, expect, it, vi } from "vitest";
+import {
+  afterAll,
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  it,
+  vi,
+} from "vitest";
 import { get } from "svelte/store";
-import { keys } from "lamb";
-import { Gas, Wallet } from "@dusk-network/dusk-wallet-js";
+import { ProfileGenerator } from "$lib/vendor/w3sper.js/src/mod";
+import { generateMnemonic } from "bip39";
 
-import { addresses, transactions } from "$lib/mock-data";
-import { rejectAfter, resolveAfter } from "$lib/dusk/promise";
+import walletCache from "$lib/wallet-cache";
+import { getSeedFromMnemonic } from "$lib/wallet";
 
 import { walletStore } from "..";
-import { waitFor } from "@testing-library/svelte";
+import { getKey } from "lamb";
 
-const settleTime = 1000;
-
-vi.useFakeTimers();
-
-describe("walletStore", async () => {
-  const balance = { maximum: 100, value: 1 };
-  const wallet = new Wallet([]);
-
-  const defaultSyncOptions = {
-    from: undefined,
-    onblock: expect.any(Function),
-    signal: expect.any(AbortSignal),
-  };
-
-  const abortControllerSpy = vi.spyOn(AbortController.prototype, "abort");
-  const blockHeightSpy = vi
-    .spyOn(Wallet, "networkBlockHeight", "get")
-    .mockResolvedValue(1536);
-  const getBalanceSpy = vi
-    .spyOn(Wallet.prototype, "getBalance")
-    .mockResolvedValue(balance);
-  const getPsksSpy = vi
-    .spyOn(Wallet.prototype, "getPsks")
-    .mockResolvedValue(addresses);
-  const historySpy = vi
-    .spyOn(Wallet.prototype, "history")
-    .mockResolvedValue(transactions);
-  const resetSpy = vi
-    .spyOn(Wallet.prototype, "reset")
-    .mockResolvedValue(void 0);
-  const stakeInfoSpy = vi
-    .spyOn(Wallet.prototype, "stakeInfo")
-    .mockResolvedValue({});
-  const stakeSpy = vi
-    .spyOn(Wallet.prototype, "stake")
-    .mockResolvedValue(void 0);
-  const syncSpy = vi.spyOn(Wallet.prototype, "sync").mockResolvedValue(void 0);
-  const transferSpy = vi
-    .spyOn(Wallet.prototype, "transfer")
-    .mockResolvedValue(void 0);
-  const unstakeSpy = vi
-    .spyOn(Wallet.prototype, "unstake")
-    .mockResolvedValue(void 0);
-  const withdrawRewardSpy = vi
-    .spyOn(Wallet.prototype, "withdrawReward")
-    .mockResolvedValue(void 0);
+describe("Wallet store", async () => {
```

Comment:
Maybe "WalletStore" instead, to be consistent "NetworkStore".

Reply:
I'll change `NetworkStore` instead: is that the one different. 

(as `NetworkStore` is not an existing file name, I should be descriptive as I was in the wallet store)